entry_points.txt

```diff
@@ -113,6 +113,7 @@ tenants = cfme.configure.access_control:TenantCollection
 time_profiles = cfme.configure.settings:TimeProfileCollection
 users = cfme.configure.access_control:UserCollection
 utilization = cfme.optimize.utilization:UtilizationCollection
+v2v_infra_mappings = cfme.v2v.infrastructure_mapping:InfrastructureMappingCollection
 v2v_mappings = cfme.v2v.migrations:InfrastructureMappingCollection
```

Comment:
Q: could you please explain having two same collections ?

Reply:
I am going to delete the old collection when all the refactored files (all Pr's )related V2V are in .
Keeping it now so that jenkins tests can still run until all PR's are merged.